examples/app/templates/batch-job.yaml

```diff
@@ -21,4 +21,5 @@ spec:
           | default .Chart.AppVersion }}
         name: pi
         resources: {}
+      initContainers: null
```

Comment:
please ignore`initContainers` in templates if it does not exist in the source manifests.

Reply:
This should be removed from this and the other templates where it was incorrect.